api/v1/cosmosfullnode_types.go

```diff
@@ -32,11 +32,21 @@ const CosmosFullNodeController = "CosmosFullNode"
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// Ordinal specifies the configuration for pod ordinal numbers
+type Ordinal struct {
+	// Start specifies the initial ordinal number for pod naming
+	// +kubebuilder:validation:Minimum:=0
+	Start int32 `json:"start,omitempty"`
```

Comment:
From StatefulSet

> start is the number representing the first replica's index. It may be used to number replicas from an alternate index (eg: 1-indexed) over the default 0-indexed names, or to orchestrate progressive movement of replicas from one StatefulSet to another. If set, replica indices will be in the range: [.spec. ordinals. start, .spec. ordinals. start + .spec. replicas). If unset, defaults to 0. Replica indices will be in the range: [0, .spec. replicas).

Reply:
Yes, the othere references in the code , configmap, pvc builders ensure that the range is  (spec.ordinals.start, spec.ordinals.start + spec.replicas)